etc/tests/test_integration.py

```diff
@@ -76,6 +76,7 @@ def test_tyk():
     endpoints = [
```

Comment:
Recommend more tests for RNAget -- though will that be possible without the HTSGet middleware?

Reply:
Yes, more RNAget tests will need to go through ingest and htsget, but they are not ready yet.